lib/simplifyExpression/collectAndCombineSearch/ConstantOrPowerTerm.js

```diff
@@ -0,0 +1,45 @@
+const NodeType = require('../../node/Type');
+
+// Returns the base if the node is on power form, else returns the node as it is constant.
+// e.g 2^4 returns 2
+// e.g 3 returns 3, as that is the node
+function getBaseNode(node) {
+  if (node.args) {
+    return node.args[0];
+  }
+  else {
+    return node;
+  }
+}
+
+// Returns the value of the base.
+function getBaseValue(node) {
+  return node.value;
+}
+
+// Returns the node that is an exponent to a constant, or a constant node with
+// value 1 if there's no exponent.
+// e.g. on the node representing 2^3, returns a constant node with value 3
+// e.g. on the node representing 2, returns a constant node with value 1 (since 2 = 2^1)
+function getExponentNode(node) {
+  if (node.args === undefined || !node.args[1]) {
+    return false;
```

Comment:
but if I return constant node 1, it will skip the addOneExponent because of (!ConstantOrPowerTerm.getExponentNode(child)). So I dont really have to return a constant node 1 here, because it will be fixed in addOneExponent.